.gitignore

```diff
@@ -1,13 +1,8 @@
-node_modules
-lib
-server
-babel-plugin
-webpack-plugin
 coverage
-yarn.lock   
+lib
+node_modules
+yarn.lock
 
-# misc
-local-scripts/
```

Comment:
Do you mind keeping this? It's handy to have some local scripts at times for testing the library in different apps without having to publish a new version

Reply:
I'm not particularly fond of keeping this in because it's very specific to your workflow. Is it possible you can do one of either:

1. Not commit the changes
2. Setup a global gitignore (see [here](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c))

If this is still inconvenient, then can we settle on a more standardised naming convention? `local-scripts` implies it might be a generated artifact when it is not. Could we use something like `*.tmp` (files, used in Jira frontend), `temp` (dir), `tmp` (dir), or `.tmp` (dir).

Reply:
I am good with tmp dir